src/praisonai/praisonai/cli/commands/validate.py

```
@@ -0,0 +1,281 @@
+"""
+YAML configuration validation command.
+
+Validates agents/tasks/workflow YAML configurations with comprehensive error checking.
+"""
+
+import sys
+from pathlib import Path
+from typing import Optional
+
+import typer
+from rich import print
```

Ruff (0.15.17)

[error] Import `print` is shadowing a Python builtin

(A004)